as2/6.0/modules/ROOT/pages/index.adoc

```diff
@@ -1,100 +1,97 @@
-You should have familiarity with AS2.
+* The AS2 target resource
+* Anypoint Studio version 7.6 or later
 
 === Considerations when Migrating from the AS2 connector 5.x
 
-AS2 Connector 6.0.0 requires Mule 4.3.0 or later. Refer to xref:as2-connector-upgrade-migrate.adoc[Upgrade and Migrate] for more information.
-
-== Audience
-
-* Starting user
-see xref:as2-connector-studio.adoc[Studio Configuration]. The
-xref:as2-connector-examples.adoc[Examples] topic provides one or more use cases for using the connector.
-* Power user - Read the xref:as2-connector-xml-maven.adoc[XML and Maven Support],
-and xref:as2-connector-examples.adoc[Examples] topics.
+AS2 Connector 6.0 requires Mule 4.3.0 or later. Refer to xref:as2-connector-upgrade-migrate.adoc[Upgrade and Migrate] for more information.
 
 == Common Use Cases For the Connector
 
 * Send with sync MDN
 
-Sends a message through HTTP or HTTPS using the AS2 protocol requesting the partner for a synchronous MDN (receipt).
+Sends a message through HTTP or HTTPS using the AS2 protocol requesting the partner for a synchronous MDN (receipt)
 
 * Send with async MDN
 
-Sends a message through HTTP or HTTPS using the AS2 protocol requesting the partner for an asynchronous MDN (receipt).
+Sends a message through HTTP or HTTPS using the AS2 protocol requesting the partner for an asynchronous MDN (receipt)
 
 * Listener
 
-Source that receives messages via HTTP or HTTPS using the AS2 protocol.
+Source that receives messages via HTTP or HTTPS using the AS2 protocol
 
 * MDN Listener
 
-Source that receives asynchronous MDN receipts via HTTP or HTTPS using the AS2 protocol.
+Source that receives asynchronous MDN receipts via HTTP or HTTPS using the AS2 protocol
 
 * Non repudiation Listener
 
-Source that receives a signed copy of a receipt or a message for persistently storing it for the non-repudiation.
+Source that receives a signed copy of a receipt or a message for persistent non-repudiation storage
 
-== Non-Repudiation Storage of AS2 messages and MDNs
+== Non-Repudiation Storage of AS2 Messages and MDNs
 
 If needed, you can configure non-repudiation storage for your AS2 operations to support regulatory or compliance requirements.
 
-The following prerequisites are for enabling non-repudiation of a message in an AS2 connector:
-
-* Only signed messages and MDNs are eligible for non-repudiation.
-
-* Users must implement and configure a separate Mule flow that acts as a non-repudiation flow.
+Only signed messages and MDNs are eligible for non-repudiation.
 
-* This flow has access to the signed mime message that is passed to it along with appropriate non-repudiation specific attributes.
+The following AS2 Connector 6.0 sources and operations support the non-repudiation feature:
 
-* It is this non-repudiation specific Mule flow’s responsibility to take, store and act upon the message and attributes it receives.
+* AS2 Send Sync Operation
+* AS2 Send Async Operation
+* AS2 Listener
+* AS2 MDN Listener
```

Comment:
```suggestion
*  *As 2 listener* source
* *As 2 mdn listener* source
* *Send with Async MDN* operation
* *Send with Sync MDN* operation
```

Reply:
Names as appear in Studio

Reply:
Here I'm not referring to the UI and also, it looks terrible in the UI.